spec/shr_questionnaire.txt

```diff
@@ -29,6 +29,7 @@ Description:    "A structured set of questions intended to guide the collection
 0..1            VersionString
 0..1            NameAsText
 0..1            Title
+                StatusOrActive is type Status
```

Comment:
StatusOrActive seems like a confounding name since `active` is a concept in the FHIR resource status value set.  Perhaps make this clearer by changing the element name to `StatusOrIsActive` in  shr_foundation.txt?  
Also, more for my curiosity, under what conditions in FHIR would we need to have a status attribute sliced to support 2 different data types for this?  Seems like both `Status` and `Active` are already designated as subclasses, so do we need `StatusOrActive` as a parent class?

Reply:
I went though about 30 resources that are "core" in FHIR (observation, condition, patient, etc.) and each had either a Status or Active attribute. There was a pattern but with several exceptions (entities like Patient and Organization tended toward Active, the remainder to Status). It just seems crazy. This was my attempt to abstract this so SHR doesn't have such arbitrary dichotomies. I like the "IsActive" suggestion. But it would be much cleaner if SHR always used coded status like "active" "inactive" instead of true/false. Unfortunately we don't have a way to map that. I'll think about how this can be improved.